docs/snippets/oracle-credentials-secret.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: oracle-secret
+  labels: 
+    type: oracle
+type: Opaque
+stringData:
+  privateKey: "**Private key value**, use |- for multiline"
+  fingerprint: "**Fingerprint value**"
```

Comment:
(optional) For the snippets, you can end all of them with an empty line just to keep them the same as the other ones :)